services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py

```diff
@@ -60,9 +61,13 @@ def _get_s3_path(s3_bucket: str, labels: dict[str, str], volume_name: str) -> Pa
     return Path(f"/{joint_key}")
 
 
-async def _read_stream(stream):
+async def _read_stream(stream: StreamReader) -> str:
+    output = ""
     while line := await stream.readline():
-        logger.info(line.decode().strip("\n"))
+        message = line.decode()
+        output += message
+        logger.debug(message.strip("\n"))
```

Comment:
Q; is it really necessary to output logs both here and in the client code? see line 132?

Reply:
I'd say yes, one is useful for debugging, if the process hangs or is taking forever there is no feedback.

The the one below in info level, is a recap at the end of the upload operation which is the only one being logged.